ha/util/machine_id.py

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env python3
```

Comment:
I think this file can be renamed and can be used to get different attributes like machine id. Can not think of other attributed now.
May be generate_uuid function can also be shifted here.

Reply:
Maybe we can rename the file when further attributes are identified; so that appropriate name can be given to it,  based on what all its going to handle. Is that okay ? 

The generated uuid is local to the file and not a generic attribute as such ; so not moving it here

Reply:
Ok. Yeah. That is fine.

Reply:
Ok. Yeah, that is fine.